infra/license/README.md

```diff
@@ -0,0 +1,54 @@
+# License Self Verification
+
+## Background
+
+Numerous kinds of Node packages are needed to build and use ONE-vscode but some packages may have unacceptable license for both ONE and ONE-vscode.
+Therefore we should review whether licenses of each package can be used or not.
+However, if we check all of them manually, we may mistake or miss something.
+To reduce such risk and labor, automatic license checker would be good solution.
+
+## Process & Architecture
+
+### Flowchart
+
+![flowchart](flowchart.png)
+
+### Terminology
+
+- license-judgment.json
+  - Each of license names are got from the result of `license-checker`.
+  - Each license are classfied to following categories
+    - `Allowed` : Allowed to use
+    - `Warning` : Manual review for each package is needed
+    - `Denied` : Not allowed to use
+- package-judgment.json
+  - Each judgment for a package is consisted of following information
+    ```json
+    "pkgname@version": {
+      "licenses": "License name",
+      "repository": "URL for package repository",
+      "ONE_permitted": "yes / conditional / no",
+      "ONE_caution": "(Optional field) Cautions when package is used"
+    },
+    ```
+    - `ONE_permitted` : Whether the package is allowed in ONE-vscode
+      - yes : Allowed to use
+      - conditional : Allowed to use under specific conditions. Needed to be checked before release.
+      - no : Not allowed to use
+    - `ONE_cuation` : Comment about why `ONE_permitted` is `conditional` or `no`.
+
+### Design Note
+
+- This flow is run on Ubuntu, Windows and MacOS
+  - Even same package is installed, dependent packages could be different
+- `PASS` and `FAIL` in the flowchart is the result of self verification
```

Comment:
The term `self verification` is confusing to me. How about `verification by CI` or `automated verification`?

Reply:
Main point of this sentence was "Even the flowchart is ended with FAIL, the Github Action would have green light because Github Action runner itself is successful".
Which sentence would be good to represent this :(
Or would it be better to change `PASS` and `FAIL` to another words...?

Reply:
To me 'self' means developer (not CI) but I guess you thought that 'self' is something done by developer after running CI. I see. I am fine with the term 'self'.

Can we add some statement at the top of this readme file?

`Each developer should verify license himself/herself, with the automated verification result from CI.`

Reply:
Hmm... I think removing `self` seems better too :D
Is it fine with following changes?
- Remove `self` word
- Add "This verification process is not official. It is just automated version of developer's license verification" to `Limitations` or top of file